### ISSUE TITLE
IX-React doesn't work with Parcel JS bundler

### ISSUE THREAD

**Issue by padibana** [siemens/ix#228]
## The issue
Trying to import any `ix-react` components in projects that use `parcel` results in an error. 

## Screenshot
![image](https://user-images.githubusercontent.com/1394209/206222201-4bb5478c-b791-45e2-8fc9-41fcfc691931.png)

## Hello World Sample Project
[ix.zip](https://github.com/siemens/ix/files/10177492/ix.zip)

### What type of frontend frameware are you seeing the problem on?

React

### Which version of iX do you use?

v1.1.1

### Code to produce this issue.

```shell
I've included a zip file with a simple hello world, but I have experienced the same issue in other projects based on parcel.
```

**Comment by danielleroux** [comment]
@padibana Its already fixed with #219 you can also test it with `1.2.0-beta.0`. The fix will be released with [1.2.0](https://github.com/siemens/ix/milestone/7)

Also do not forgot to import the css inside the index.js

```javascript
import { createRoot } from "react-dom/client";
import { App } from "./App";

// Import styling!
import "@siemens/ix/dist/siemens-ix/siemens-ix.css";

const container = document.getElementById("app");
const root = createRoot(container);

root.render(<App />);
```

**Comment by danielleroux** [comment]
Fixed #219 which is part of [1.2.0 ](https://github.com/siemens/ix/milestone/7)
